code/game/objects/items/devices/radio/radio.dm

```diff
@@ -248,7 +248,7 @@ var/global/list/default_medbay_channels = list(
 						0, "*garbled automated announcement*", src,
 						message, from, "Automated Announcement", from, "synthesized voice",
 						4, 0, list(0), connection.frequency, "states")
-	qdel(A)
+	QDEL_NULL(A)
```

Comment:
this is unnecessary, revert back to qdel. Function locals cease existing at the end of scope anyways.